Documentation/guides/messages/xa4313.md

```diff
@@ -0,0 +1,20 @@
+---
+title: Xamarin.Android error/warning XA4313
+description: XA4313 error/warning code
+ms.date: 03/19/2021
+---
+# Xamarin.Android error/warning XA4313
+
+The specified Framework assembly has been deprecated.
+You need up upgrade to use the specified NuGet Package.
+
+For example, to upgade `OpenTK-1.0` remove
+
+    '<Reference Include="OpenTK-1.0" />'
+
+from your csproj then add
+
+    '<PackageReference Include="Xamarin.Legacy.OpenTK" Version="1.0.0" />'
```

Comment:
the version should probably be `0.0.1-alpha` ?

Reply:
I'll probably push a release 1.0.0 before we ship this.